### DISCUSSION TITLE
Problems with Windows 10 UTM on iPad

### DISCUSSION THREAD

**Discussion by ghost** [utmapp/UTM#6601] [UTM Help (iOS)]
I downloaded the windows 10 iso file here: [https://www.microsoft.com/en-au/software-download/windows10](https://www.microsoft.com/en-au/software-download/windows10), and I leaved all the settings as default.

I am constantly getting this message: **The data couldnât be read because it isnât in the correct format.**
Occasionally I get into the device, but I always get stuck on startup.nsh.
I tried pressing every key, even esc, and it always goes to **Shell>**

**Configuration (required)**
* UTM Version: 4.5.3
* OS Version: Windows 10
* Device Model: Standard PC (Q35 + ICH9, 2009) (alias of pc-q35-7.2) (q35)
* My device: iPad 8th gen (iPadOS 17.5.1)
* Is it jailbroken (name jailbreak used)? I used AltStore
* How did you install UTM? Via AltStore source

[**Debug log when I did get in**](https://pastebin.com/bYqZKkce)

**Comment by ghost** [comment]
I tried again, and finally I got to the next stage, where it said something about not having enough memory to do something.

I added more memory and tried again, but it wouldn’t work.
Here is an image of very first stage that disappears quickly.
![IMG_9696](https://github.com/user-attachments/assets/1b987779-87f3-4454-a93e-df458f7c8c87)

**Comment by happysmaran** [comment]
As far as I remember, it mentions on the wiki that no more than 20% of the iPhone/iPad memory can be dedicated, otherwise the OS will terminate the process. So I'm afraid that Windows 10 cannot work, since it wants more but cannot give it. Windows 7/XP might?
